be/src/agent/task_worker_pool.h

```diff
@@ -201,4 +202,21 @@ class ReportWorkgroupTaskWorkerPool : public TaskWorkerPool<AgentTaskRequestWith
     }
 };
 
+class ReportResourceUsageTaskWorkerPool : public TaskWorkerPool<AgentTaskRequestWithoutReqBody> {
+public:
+    ReportResourceUsageTaskWorkerPool(ExecEnv* env, int worker_num) : TaskWorkerPool(env, worker_num) {
+        _callback_function = _worker_thread_callback;
+    }
+
+private:
+    static void* _worker_thread_callback(void* arg_this);
```

Comment:
(void * const arg_this)

Reply:
All the subclasses of `TaskWorkerPoolBase` use `typedef void* (*CALLBACK_FUNCTION)(void*)`, so changing it to `void * const arg_this` need modify all the subclasses.

BTW, `void* const` for an argument is meaningless. Even if `arg_this` is `void *` not `void* const`, and is changed in `_worker_thread_callback`, it hasn't side-effect.